cmp/relativetimestamp/RelativeTimestamp.js

```diff
@@ -102,6 +114,8 @@ export const relativeTimestamp = elemFactory(RelativeTimestamp);
  * @param {(Date|int)} timestamp - Date object or milliseconds that will be used as reference for this component
  * @param {Object} [options]
  * @param {boolean} [options.allowFuture] - Allow dates greater than Date.now()
+ * @param {string} [options.prefix] - Label preceding timestamp
+ * @param {int} [options.prefixMargin] - Right margin in pixels for the prefix label
```

Comment:
Could we just have a single space between the prefix and formatted string? That would adjust with the font size and not be something we need to specify / default in pixels.  I was thinking it would naturally be equivalent to the suffix. Any reason to treat them differently?

I also don't want to change the output of the getRelativeTimestamp method itself - that should continue to return a string (as documented and generally expected).